web/src/lib/domain-logic/essentialQuestions.ts

```diff
@@ -0,0 +1,155 @@
+import { EssentialQuestionsProps, ProfileLabelField } from "@/lib/types/types";
+import {
+  emptyIndustrySpecificData,
+  Industry,
+  IndustrySpecificData,
+  LookupIndustryById,
+  ProfileData,
+} from "@businessnjgovnavigator/shared";
+
+type EssentialQuestionFunction = (industry: Industry) => boolean;
+interface EssentialQuestion
+  extends Omit<EssentialQuestionsProps<ProfileData[keyof ProfileData]>, "fieldName" | "choices"> {
+  fieldName: keyof IndustrySpecificData;
+  validationFunc: EssentialQuestionFunction;
```

Comment:
"validationFunc" that returns a boolean is a poor name because it doesn't tell me anything about what it does. Does it return true if it's valid? True if there are validation errors and it is invalid? A clearer name could help with readability

Reply:
Changed my mind... I looked below and it doesn't seem like this validation has anything to do with the user data? It's testing if the essential question is applicable to the industry? This seems the same as "conveniencFunc"? Can you explain the difference here, I think I am missing it